src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    static final Random RANDOM = new Random();
```

Comment:
```suggestion
    private final Random random = new Random();
```

Reply:
Tell me please in the class Lottery. I have variable random. I need create object there too?  private final Random random = new Random();